social/author/templates/author/edit.html

```diff
@@ -0,0 +1,37 @@
+<html>
+<body>
+
+<form action="/a/edit_post/" method="POST">
+{% csrf_token %}
+  First name:<br>
```

Comment:
You can pass in a form and use {{ form }} and Django will render the form for you. For future reference.